### DISCUSSION TITLE
Dingtian relay has "Jogging" feature which turns relay on and off for a short moment. Is this supported in Cerbo?

### DISCUSSION THREAD

**Discussion by onnimonni** [Lucifer06/RemoteGPIO#48] [Q&A]
I'm using the Dingtian smart relays to control stupid but high voltage relays which need a short electric signal to be toggled on or off.

The Dingtian relay has this jogging feature which works perfectly for my use case.

<img width="1259" alt="Screenshot 2024-11-26 at 12 47 45" src="https://github.com/user-attachments/assets/65cf6e8d-7865-4367-b8a5-1c50d1df3e1e">

How could I achieve something similiar from Cerbo?

I think the RemoteGPIO and even the default Cerbo relays don't have this.

Would you @Lucifer06 know anything about this?

**Comment by onnimonni** [comment]
I saw this question in the Victron community forum:
https://communityarchive.victronenergy.com/questions/195986/cerbo-gx-relais-to-give-a-short-pulse-instead-of-t.html

And it seems that this has not been yet possible. At least nobody else hasn't yet answered it.

**Comment by Lucifer06** [comment]
Sorry for the delay....
I think what you are asking for is doable just with configuration, but it would consume 2x relays for one function:
Let's assume Relay 1 is controlled by RemoteGPIO, in a normal manner with 2x states; Closed or open
The output of Relay1 would be wired to Input2 on the Dingtian.
In Dingtian web interface, go in Input Link Relay, and configure the Input 2 so it generates a toggle on the Relay2
I haven't tried it myself, but pretty sure it will work. Please report here how it goes.
